brainiak/io.py

```diff
@@ -59,6 +62,10 @@ def load_images_from_dir(in_dir: Union[str, Path], suffix: str = "nii.gz",
     files = sorted(in_dir.glob("*" + suffix))
     for f in files:
         yield nib.load(str(f))
+        logger.info(
+            'file %s is read',
+            f
+        )
```

Comment:
done, but I would put some brief `logger info` to indicate in the beginning of some stages (reading file, apply masking, separating epochs, etc.)